SpatialGDK/Source/SpatialGDK/Private/SpatialView/ViewDelta/ViewDelta.cpp

```diff
@@ -631,12 +462,12 @@ Worker_ComponentSetAuthorityChangeOp* ViewDelta::ProcessEntityAuthorityChanges(W
 	}
 }
 
-ViewDelta::ReceivedEntityChange* ViewDelta::ProcessEntityExistenceChange(ReceivedEntityChange* It, ReceivedEntityChange* End,
-																		 EntityDelta& Delta, bool bAlreadyInView, EntityView& View)
+FReceivedEntityChange* ViewDelta::ProcessEntityExistenceChange(FReceivedEntityChange* It, FReceivedEntityChange* End, EntityDelta& Delta,
+															   bool bAlreadyInView, EntityView& View)
 {
 	// Find the last element relating to the same entity.
 	const Worker_EntityId EntityId = It->EntityId;
-	It = std::find_if(It, End, DifferentEntity{ EntityId }) - 1;
+	It = std::find_if(It, End, FDifferentEntity{ EntityId }) - 1;
```

Comment:
Since we are using Unreal abstractions, replace with Algo::FindByPredicate ?

Reply:
I don't think find by predicate searches between two iterators.